### ISSUE TITLE
New release in the near future

### ISSUE THREAD

**Issue by jarus** [vmware-archive/salty-vagrant#88]
Hi,
thanks for salty-vagrant but will be there a new release in the near future. This would add support of vagrant boxes with Debian Wheezy.

**Comment by akoumjian** [comment]
Hi Jarus,

I do apologize. We are trying to get the plugin merged into vagrant proper (mitchellh/vagrant#1626). However, I will put out a plugin release with bootstrap 1.5.5 tonight.

@raadad, we need to figure out a canonical way to develop both the native vagrant plugin and the separate plugin in parallel, so that we can continue to push new releases once we get it merged in (without having to wait for vagrant to include them).

**Comment by jarus** [comment]
@akoumjian Thank you very much.

A included version of salty into vagrant would be really nice. :thumbsup: 
Maybe salty could download the last stable version of bootstrap to make it obsolet to release a new version of salty which only update the bootstrap script.

**Comment by akoumjian** [comment]
First thing to note is that you can use whatever version of the bootstrap script you want already. Just download it and mark it in your Vagrantfile using the bootstrap_script option: https://github.com/saltstack/salty-vagrant#other

I have considered using a model which automatically updates the bootstrap script, but have decided against it. Part of the problem is that even the stable branch of the bootstrap script sometimes breaks, and if a user finds a setup that works, I don't want it breaking unexpectedly.

**Comment by raadad** [comment]
@akoumjian 
I am happy to manually push changes to the branch until it gets merged. just shoot me a mail, If I dont merge any changes soon enough

**Comment by raadad** [comment]
Latest changes have been merged into my PR on vagrant: https://github.com/mitchellh/vagrant/pull/1626
If anyone finds the time to test this again it would be appreciated.

@akoumjian my current workflow is to:
pull the latest salty vagrant, overwrite all files that are used in the vagrant provisioner, then reverse hunks of area's that were changed for the plugin to work as a built in provisioner.

This takes about 10 minutes to do, but then testing takes another 15 minutes, so it seems manageable thing to do until it gets merged

**Comment by akoumjian** [comment]
Closing this, as we are now merged into core and working on updating that release.
